cmd/world/cardinal/start.go

```diff
@@ -33,8 +33,8 @@ const (
 var (
 	// ValidLogLevels Valid log levels for zerolog
 	validLogLevels = strings.Join([]string{
-		zerolog.DebugLevel.String(),
```

Comment:
Same as my comment in the linear task, the order doesn't matter. If we're ordering it, we should follow zerolog's log levels: trace, debug, info, warn, error, fatal, panic. `disabled` should be fine at either the first or last in the order.

Reply:
adjusted